packages/base/cards-grid.gts

```diff
@@ -1,15 +1,49 @@
 import { action } from '@ember/object';
 import { on } from '@ember/modifier';
+import { tracked } from '@glimmer/tracking';
 import { restartableTask } from 'ember-concurrency';
 import { Component, Card, relativeTo } from './card-api';
 import { CardContainer, IconButton } from '@cardstack/boxel-ui';
-import { chooseCard, catalogEntryRef } from '@cardstack/runtime-common';
+import {
+  chooseCard,
+  catalogEntryRef,
+  getCards,
+  baseRealm,
+} from '@cardstack/runtime-common';
 import { type CatalogEntry } from './catalog-entry';
 
 class Isolated extends Component<typeof CardsGrid> {
   <template>
-    <CardContainer class='demo-card cards-grid' @displayBoundaries={{true}}>
-      This cards-grid instance should become even better.
+    <CardContainer class='cards-grid'>
+      <ul class='cards-grid__cards'>
+        {{#each this.request.instances as |card|}}
+          <li data-test-cards-grid-item={{card.id}}>
+            <CardContainer class='grid-card'>
+              <div class='grid-card__thumbnail'>
+                <div
+                  class='grid-card__thumbnail-text'
+                  data-test-cards-grid-item-thumbnail-text
+                >{{card.constructor.displayName}}</div>
```

Comment:
Consider using https://github.com/cardstack/boxel/blob/4251057f5a085e6c671956fb2092f59809e80497/packages/host/app/helpers/card-type-display-name.ts

Reply:
This means moving this helper into runtime-common, though. To be able to use both from host and from base

Reply:
Seems appropriate in this case.